awpy/types.py

```diff
@@ -666,21 +666,20 @@ class RoundStatistics(TypedDict):
 
 
 # Type to represent tile id for navigation tiles.
-# TODO: Incorporate TileIdObject into rest of repo.
-TileIdObject = NewType("TileIdObject", int)
+# TODO: Incorporate TileId into rest of repo.
+TileId = NewType("TileId", int)
 
 # Return type for awpy.analytics.map_control._bfs_helper.
 # Contains map control values for one team.
-# Maps TileIdObject to list of tile map control values.
-TeamMapControlValuesDict = NewType(
-    "TeamMapControlValuesDict", dict[TileIdObject, list[float]]
-)
+# Maps TileId to list of tile map control values.
+TeamMapControlValues = NewType("TeamMapControlValues", dict[TileId, list[float]])
 
 # Return type for awpy.analytics.map_control.graph_to_tile_neighbors
-# Maps TileIdObject to set of neighboring tiles.
-TileNeighborsDict = NewType("TileNeighborsDict", dict[TileIdObject, set[int]])
+# Maps TileId to set of neighboring tiles.
+TileNeighbors = NewType("TileNeighbors", dict[TileId, set[int]])
 
 
+@dataclass
 class TileDistanceObject(TypedDict):
     """Dataclass with data for map control tile distance calculations.
 
```

Comment:
If we are making this a data class (which i think we should) it shouldnt inherit from TypedDict. And subsequently should be constructed as a class and accessed via `my_tile_distance_object.tile_id`.

Same for the other cases.

Reply:
Yup addressed in next commit for branch. Thanks for the suggestions!